### PR TITLE
sig-security: add draft agenda for 2017-06-07 meeting

### PR DESCRIPTION
**Pull request by riyazdf**
Will update more as I post to forum, get more agenda input.  cc @tych0 

Signed-off-by: Riyaz Faizullabhoy <riyaz.faizullabhoy@docker.com>

 </img>

### CODE REVIEW

File: reports/sig-security/2017-06-07.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Meeting location: https://docker.zoom.us/j/779801882
+
+Announcement: Moby project forum post coming soon!
+
+Previous meeting notes: [2017-05-24](2017-05-24.md)
+
+## Agenda
+- Introductions & Administrivia
+- RFC on `"probational"` channel
```
Comment (tych0) [inline reports/sig-security/2017-06-07.md]: Maybe list the other names that were given as suggestions as well?

Comment (riyazdf) [inline reports/sig-security/2017-06-07.md]: sounds good!

Comment (riyazdf) [inline reports/sig-security/2017-06-07.md]: done.  Let me know if I missed any of the suggestions

**Comment by tych0** [comment]
LGTM!

**Pull request merged**